libraries/AC_AutoTune/AC_AutoTune_Heli.cpp

```diff
@@ -0,0 +1,1200 @@
+#define AUTOTUNE_FFI_RATIO_FOR_TESTING     0.5f     // I is set 2x smaller than VFF during testing
+#define AUTOTUNE_FFI_RATIO_FINAL           0.5f     // I is set 0.5x VFF after testing
+#define AUTOTUNE_PI_RATIO_FINAL            1.0f     // I is set 1x P after testing
+#define AUTOTUNE_YAW_PI_RATIO_FINAL        0.1f     // I is set 1x P after testing
+#define AUTOTUNE_RD_STEP                  0.0005f     // minimum increment when increasing/decreasing Rate D term
+#define AUTOTUNE_RP_STEP                  0.005f     // minimum increment when increasing/decreasing Rate P term
+#define AUTOTUNE_SP_STEP                  0.05f     // minimum increment when increasing/decreasing Stab P term
+#define AUTOTUNE_PI_RATIO_FOR_TESTING      0.1f     // I is set 10x smaller than P during testing
+#define AUTOTUNE_RD_MAX                  0.020f     // maximum Rate D value
+#define AUTOTUNE_RLPF_MIN                  1.0f     // minimum Rate Yaw filter value
+#define AUTOTUNE_RLPF_MAX                  20.0f     // maximum Rate Yaw filter value
+#define AUTOTUNE_RP_MIN                   0.001f     // minimum Rate P value
+#define AUTOTUNE_RP_MAX                   0.3f     // maximum Rate P value
+#define AUTOTUNE_SP_MAX                    10.0f     // maximum Stab P value
+#define AUTOTUNE_SP_MIN                    3.0f     // maximum Stab P value
+#define AUTOTUNE_RFF_MAX                   0.5f     // maximum Stab P value
+#define AUTOTUNE_RFF_MIN                   0.025f    // maximum Stab P value
+#define AUTOTUNE_D_UP_DOWN_MARGIN          0.2f     // The margin below the target that we tune D in
+
+#define AUTOTUNE_SEQ_BITMASK_VFF             1
+#define AUTOTUNE_SEQ_BITMASK_RATE_D          2
+#define AUTOTUNE_SEQ_BITMASK_ANGLE_P         4
+#define AUTOTUNE_SEQ_BITMASK_MAX_GAIN        8
+
+const AP_Param::GroupInfo AC_AutoTune_Heli::var_info[] = {
+
+    // @Param: AXES
+    // @DisplayName: Autotune axis bitmask
+    // @Description: 1-byte bitmap of axes to autotune
+    // @Bitmask: 0:Roll,1:Pitch,2:Yaw
+    // @User: Standard
+    AP_GROUPINFO("AXES", 1, AC_AutoTune_Heli, axis_bitmask,  7),  // AUTOTUNE_AXIS_BITMASK_DEFAULT
+
+    // @Param: SEQ
+    // @DisplayName: AutoTune Sequence Bitmask
+    // @Description: 2-byte bitmask to select what tuning should be performed.  Max gain automatically performed if Rate D is selected. Values: 7:All,1:VFF Only,2:Rate D Only,4:Angle P Only,8:Max Gain Only,3:VFF and Rate D (incl max gain),5:VFF and Angle P,13:VFF max gain and angle P
+    // @Bitmask: 0:VFF,1:Rate D,2:Angle P,3:Max Gain Only
+    // @User: Standard
+    AP_GROUPINFO("SEQ", 2, AC_AutoTune_Heli, seq_bitmask,  5),
+
+    // @Param: MIN_FRQ
+    // @DisplayName: AutoTune minimum sweep frequency
+    // @Description: Defines the start frequency for sweeps and dwells
+    // @Range: 10 30
+    // @User: Standard
+    AP_GROUPINFO("MIN_FRQ", 3, AC_AutoTune_Heli, min_sweep_freq,  10.0f),
```

Comment:
I'd prefer if the "MIN" or "MAX" went at the end of the variable name instead of in the middle.

Reply:
done